Lib/fontTools/cffLib/specializer.py

```diff
@@ -47,7 +47,10 @@ def programToCommands(program):
 					       'endchar'}:
 			parity = token in {'hmoveto', 'vmoveto'}
 			if (len(stack) % 2) ^ parity:
-				width = stack.pop(0)
+				try:
```

Comment:
For width, since it's optional, I'll just change the if line like this:
```
if stack and (len(stack) % 2) ^ parity:
```
Then you wouldn't need the try except.